msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Receive all channel messages with RSC
+author: surbhigupta12
+description: Receive all channel messages with RSC permissions
+ms.topic: conceptual
+localization_priority: Normal
+---
+
+# Receive all channel messages with RSC
+
+> [!NOTE]
+> This feature is currently available in public developer preview only.
+
+The resource-specific consent (RSC) permissions model, originally developed for Teams Graph APIs, is now extended to bot scenarios.
+
+Currently, bots can only receive user channel messages when they are @mentioned. Using RSC, you can now request team owners to consent for a bot to receive user messages across standard channels in a team without being @mentioned. This capability is enabled by specifying the `ChannelMessage.Read.Group` permission in the manifest of an RSC enabled Teams app. After configuration, team owners can grant consent during the app installation process.
+
+For more information about enabling RSC for your app, see [resource-specific consent in Teams](/microsoftteams/platform/graph-api/rsc/resource-specific-consent#update-your-teams-app-manifest).
+
+## Enable bots to receive all channel messages
+
+The `ChannelMessage.Read.Group` RSC permission is extended to bots. With user consent, this permission allows graph applications to get all messages in a conversation and bots to receive all channel messages without being @mentioned.
+
+## Update app manifest
+
+For your bot to receive all channel messages, RSC must be configured in the Teams app manifest with the `ChannelMessage.Read.Group` permission specified in the `webApplicationInfo` property.
+
+![Update app manifest](~/bots/how-to/conversations/Media/appmanifest.png)
+
+The following is an example of the `webApplicationInfo` object:
+
+* **id**: Your Azure Active Directory (AAD) app ID. This can be the same as your bot ID.
+* **resource**: Any string. This field has no operation in RSC, but must be added and have a value to avoid error response.
+* **applicationPermissions**: RSC permissions for your app with `ChannelMessage.Read.Group` must be specified. For more information, see [resource-specific permissions](/microsoftteams/platform/graph-api/rsc/resource-specific-consent#resource-specific-permissions).
+
+The following code provides an example of the app manifest:
+
+```json
+"webApplicationInfo": {
+"id": "XXxxXXXXX-XxXX-xXXX-XXxx-XXXXXXXxxxXX",
+"resource": "https://AnyString",
+"applicationPermissions": [
+"ChannelMessage.Read.Group"
+    ]
+  }
+```
+
+The following image illustrates how to create the zip file for the app manifest:
```

Comment:
Let's remove this along with the image below it. Developers should already be familiar with how to create a zip from the manifest